common/utils/data.utils.ts

```diff
@@ -177,4 +186,5 @@ export const fetchData = async (): Promise<void> => {
   const res = await fetch(databaseURL);
   const arrBuffer = await res.arrayBuffer();
   parseData(arrBuffer);
+  console.log(topics);
```

Comment:
```suggestion
```

Reply:
Looks like there's something wrong with the ESLint config (for `common` maybe?) as this wasn't caught